src/modals/SettingsModal.tsx

```diff
@@ -370,51 +371,42 @@ export default function SettingsModal({ show, onHideModal }) {
         <Form.Group controlId="faceDisplay">
           <Form.Check
             custom
-            checked={showFace}
+            checked="true"
```

Comment:
is this intentional?

Reply:
Yes, until we know what to do with that checkbox